src/main/java/com/mycompany/myapp/slug/domain/JHLiteExtensionFeatureSlug.java

```diff
@@ -0,0 +1,18 @@
+package com.mycompany.myapp.slug.domain;
+
+import tech.jhipster.lite.module.domain.resource.JHipsterFeatureSlugFactory;
+
+public enum JHLiteExtensionFeatureSlug implements JHipsterFeatureSlugFactory {
```

Comment:
Following [this discussion](https://github.com/jhipster/jhipster-lite-extension/issues/5#issuecomment-2155117393), this should now be 
```suggestion
public enum JHLiteSampleExtensionFeatureSlug implements JHipsterFeatureSlugFactory {
```
(ditto for JHLiteSampleExtensionModuleSlug)

Reply:
maybe it should be generated by default, by the jhlite custom too?

Reply:
Which is going to be the default enum for the feature? And it would be nice if the jhlite custom talk about it in the documentation too 👍